team-tg/rnm-k8s.yaml

```diff
@@ -0,0 +1,52 @@
+# A testGrid Job 
+version: '0.9'
+emailToList: "kasung@wso2.com,edussuriyac@wso2.am,nishika@wso2.com"
+infrastructureConfig:
+  iacProvider: KUBERNETES
+  infrastructureProvider: GKE
+  containerOrchestrationEngine: None
+  includes:
+    - CentOS-7.5
+    - MySQL-5.7
+    - OPEN_JDK8
+  provisioners:
+    - name: kubernetes woring environment
+      description: Provision Infra for running intg tests
+      remoteRepository: "git@github.com:edussuriyac/wso2_testgrid_kubernetes.git"
+      scripts:
+        - name: 'Test-Phase-1-Deployment'
+          description: ''
+          type: KUBERNETES
+          phase: CREATE
+        - name: 'kubernetes-destroy-environment'
+          type: KUBERNETES
+          phase: DESTROY
+deploymentConfig:
+  deploymentPatterns:
+    - name: 'deployment'
+      remoteRepository: "git@github.com:edussuriyac/wso2_testgrid_kubernetes.git"
+      description: 'deployment of the resources in the gke'
+      scripts:
+        - name: 'deploy1'
+          type: SHELL
+          file: deploymentRepository/wso2am.sh
+        - name: 'deploy'
+          type: KUBERNETES
+          inputParameters:
+            LogFileLocations:
+            - { deploymentname: wso2apim-with-analytics-apim , containername: wso2apim-with-analytics-apim-worker , path: /home/wso2carbon/wso2am-2.6.0/repository/logs }
+            deploymentYamlFiles: 'deployment.yaml'
+            deployments: 'wso2apim-with-analytics-apim'
+            exposedDeployments : wso2apim-with-analytics-apim
```

Comment:
Do we need both 'deployments' and 'exposedDeployments'?

Reply:
resolved. Removed unnecessary yaml property